radeon-profile/ioctl_amdgpu.cpp

```diff
@@ -77,8 +77,7 @@ bool amdgpuIoctlHandler::getCoreClock(int *data) const {
 #ifdef AMDGPU_INFO_SENSOR_GFX_SCLK
     return getSensorValue(data, sizeof(data), AMDGPU_INFO_SENSOR_GFX_SCLK);
 #else
-    Q_UNUSED(data);
-    return false;
+    return getClocks(data, NULL);
```

Comment:
Reverted in https://github.com/marazmista/radeon-profile/pull/55/commits/a1e6b27e1413145691cd8e1d2afe9109505ece1a